tap_surveymonkey/mode.py

```diff
@@ -121,6 +123,9 @@ def sync_survey_details(config, state):
     }
     surveys = sm_client.make_request('surveys', params=params, state=state)
     while True:
+        if not surveys:
+            LOGGER.critical("Resource not found")
+            return None
```

Comment:
Were you able to trigger this `return`? I'm curious if we need to raise an error here instead of just returning `None`. I want the `@singer.utils.handle_top_exception(LOGGER)` in `__init__.py` to do something

Reply:
Changed to raise exception and the init can catch it.